src/include/ipc/dai-intel.h

```diff
@@ -192,8 +196,8 @@ struct sof_ipc_dai_dmic_params {
 	/* reserved for future use */
 	uint32_t reserved[5];
 
-	/**< variable number of pdm controller config */
-	struct sof_ipc_dai_dmic_pdm_ctrl pdm[0];
+	/**< Fixed number of pdm controller config */
+	struct sof_ipc_dai_dmic_pdm_ctrl pdm[SOF_DAI_INTEL_DMIC_NUM_CTRL];
```

Comment:
Whats the rational behind this change, it's missing from commit message.

Reply:
Yep, need to add.

There's fair amount of comments on kernel side and I'll do the same updates for firmware too.

Reply:
Commit text is now updated and header file comments are in sync with kernel.

Reply:
Sorry, diffing revealed a difference of one "." so new version again. There's also lot of other changes but I'll complete my part.

Reply:
Done. Finally same DMIC configuration headers as in kernel PR version.